src/auth/signup.rs

```diff
@@ -21,7 +21,7 @@ pub(crate) struct SignupInfo<'r> {
 // TODO, signup is available only when not logged in
 // if signup sucessfully, redirect to login page. (It won't log in automatically)
 // Otherwise, return Status::BadRequest and a string indicating the error. (It is not fancy at all :< )
-#[post("/api/auth/user", data = "<signup_info>")]
+#[post("/api/auth/signup", data = "<signup_info>")]
```

Comment:
This path is correct. Please refer [api](https://app.swaggerhub.com/apis-docs/CHODEHIRGD157842/Rudrist/1.0.0#/admin/post_api_auth_user).